source/MaterialXContrib/MaterialXMaya/resources/SR_brass1.xml

```diff
@@ -293,7 +293,7 @@ float mx_microfacet_sheen_NDF(float cosTheta, float roughness)
 // LUT for sheen directional albedo. 
 // A 2D table parameterized with 'cosTheta' (cosine of angle to normal) on x-axis and 'roughness' on y-axis.
 #define SHEEN_ALBEDO_TABLE_SIZE 16
-uniform float u_sheenAlbedo[SHEEN_ALBEDO_TABLE_SIZE*SHEEN_ALBEDO_TABLE_SIZE] = float[](
+const float u_sheenAlbedo[SHEEN_ALBEDO_TABLE_SIZE*SHEEN_ALBEDO_TABLE_SIZE] = float[](
```

Comment:
Isn't this directory actually obsolete? I guess these were used when we were debugging the plugin.

Reply:
On other words, should it be removed?

Reply:
Cool, I'll do this shortly.